ui/src/flux/components/FuncArgsPreview.tsx

```diff
@@ -41,23 +41,18 @@ export default class FuncArgsPreview extends PureComponent<Props> {
     const {func} = this.props
     const {args} = func
 
-    return args.map((arg, i): JSX.Element => {
-      if (!arg.value) {
-        return
-      }
+    return args.filter(a => a.value).map((arg, i) => {
+      let argValue = String(arg.value)
 
-      const separator = i === 0 ? null : ', '
```

Comment:
Why move this out of a variable, I think it makes the JSX a little harder to read below to have it inline.

Reply:
I thought it was confusing that there was another `, ` used nearby that wasn't consuming the `separator` variable. I'll move it back and clarify the naming.